server/views/layout.njk

```diff
@@ -29,13 +29,14 @@
 {% endblock %}
 
 {% block beforeContent %}
-  {{ govukPhaseBanner({
-    tag: {
-      text: "beta"
-    },
-    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
-  }) }}
-
+  <div id="phase-banner" role="complementary">
```

Comment:
instead of adding this wrapper, you should be able to add the role attribute to the phase banner macro, for example,

```
    {{ govukPhaseBanner({
      attributes: {
        role: 'complementary'
      },
      tag: {
        text: "beta"
      },
      html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
    }) }}
```

Reply:
just looking now, thanks.

Reply:
Yes. That works better. Pushing shortly.